scene/3d/reflection_probe.cpp

```diff
@@ -183,6 +183,17 @@ ReflectionProbe::UpdateMode ReflectionProbe::get_update_mode() const {
 	return update_mode;
 }
 
+void ReflectionProbe::set_importance(int p_importance) {
+	ERR_FAIL_INDEX(p_importance, 100); // should be between 0 - 100
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(p_importance < 0 || p_importance > 100, "Reflection probe priority must be between 0 and 100 (inclusive)".); // should be between 0 - 100
```

Reply:
I did change this already but yeah like this may work well also. Annoying that github is not updating the commit atm.

Reply:
btw, as we were discussing this in chat, changing this to a range of -128 to 127 might allow some more flexibility, it will mean setting the starting value in the shader to -128 which will trigger an extra clear of the accumulation variables, doubt that will be a problem but still

Reply:
I would prefer having more flexibility indeed, as not being able to make a single probe less important than others isn't feasible unless you raise priority on *all* other probes in the scene right now. How much of a performance does to the extra clear have?